### DISCUSSION TITLE
Specific panel for character animation like Toon boom

### DISCUSSION THREAD

**Discussion by sjsheelan** [opentoonz/opentoonz#6372] [Ideas]
### Is your feature request related to a problem? Please describe

Need more options and panels for character animation 

### Describe the solution you'd like

Like adobe character animator I need a separate plugin or a panel for opentoonz

**Comment by RodneyBaker** [comment]
It shouldn't take exceedingly long to implement this request but it'll be good to provide a little more detail on those things you want to see added.  Or is this just a matter of exactly replicating Toon Boom and Adobe Character Animator?  :)

Can you provide some of the character setups you use in those programs as a starting place?
Short of that can you provide a few links and examples?

**Comment by sjsheelan** [comment]
It is like the linked video

https://youtu.be/nZftheK64KE
